qcut/docs/task/invite-test/testers.md

```
@@ -0,0 +1,194 @@
+2. Add tester email to `PAYMENTS_EMAIL_ALLOWLIST`
+3. Tester can buy top-up packs or upgrade via Stripe checkout
+
+Option C — **Direct SQL** (if admin API not deployed yet):
+```sql
+UPDATE licenses SET plan = 'pro', max_devices = 3 WHERE user_id = '<id>';
+UPDATE credit_balances SET plan_credits = 500 WHERE user_id = '<id>';
+```
+
+## Database Schema (Key Tables)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)